arangod/IResearch/IResearchDataStore.cpp

```diff
@@ -1476,10 +1285,9 @@ Result IResearchDataStore::initDataStore(
   _dataStore.storeSnapshot(std::make_shared<DataSnapshot>(
       std::move(reader), std::move(engineSnapshot)));
 
-  std::string name = absl::StrCat("flush subscription for ArangoSearch index '",
-                                  index().id().id(), "'");
-  _flushSubscription = std::make_shared<IResearchFlushSubscription>(
-      _dataStore._recoveryTickLow, name);
+  if (pathExists) {
```

Comment:
why only if `pathExists`?

Reply:
Otherwise it will be created in syncIndexOnCreate after creating index will be finished